engine/src/flutter/impeller/display_list/dl_golden_blur_unittests.cc

```diff
@@ -227,26 +227,34 @@ TEST_P(DlGoldenTest, ShimmerTest) {
   EXPECT_TRUE(average_rmse >= 0.0) << "average_rmse: " << average_rmse;
 }
 
-TEST_P(DlGoldenTest, StrokedRRectFastBlur) {
```

Comment:
I don't think you actually want to remove this test...

Reply:
Doh, you're right.  This was a bad merge.

Reply:
brought it back